intelmq/bots/collectors/github_api/_collector_github_api.py

```diff
@@ -23,16 +22,15 @@
 
 
 class GithubAPICollectorBot(CollectorBot):
-    basic_auth_username = None
-    basic_auth_password = None
+    personal_access_token: str = None
```

Comment:
None is not a string
```suggestion
    personal_access_token: Optional[str] = None
```